src/main/java/athena/commands/EditCommand.java

```diff
@@ -14,9 +15,9 @@
     private Importance taskImportance;
     private String taskNotes;
 
-    public EditCommand(int index, String name, String startTime, String duration, String deadline,
+    public EditCommand(int number, String name, String startTime, String duration, String deadline,
```

Comment:
found it a bit weird that the argument order is not the same as add task, I know that the number is a required argument so its in the front, but it might look a bit messy to others.

I would have done the same, but am also not too sure about the correctness

Reply:
How about moving `number` to the first argument for `addTask` as well?

Reply:
Sounds good!